<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutoSubmit v0.5</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="windowTitle">
     <string>AutoSubmit by ProtossManse</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Submit</string>
     </attribute>
     <widget class="QPushButton" name="seedbutton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Seed</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Seed: </string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="version">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1.16.1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other (Not Supported)</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="igtMin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>00</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IGT:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Real Time(Manual):</string>
      </property>
     </widget>
     <widget class="QComboBox" name="seedType">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string>RSG</string>
      </property>
      <item>
       <property name="text">
        <string>RSG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SSG</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>130</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>m</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="igtSec">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>00</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>131</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="igtPoint">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>000</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="rtSec">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>00</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="rtPoint">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>000</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>180</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>m</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rtMin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>00</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>181</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ytLink">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Video Link(Manual)</string>
      </property>
     </widget>
     <widget class="QComboBox" name="diffiBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Easy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hard</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hardcore</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="Mods">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>72</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string>Vanilla</string>
      </property>
      <item>
       <property name="text">
        <string>Vanilla</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CaffeineMC</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="f3Box">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>F3</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="f3Button">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>0</y>
        <width>1</width>
        <height>1</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="resetButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh
(Esc)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QPushButton" name="pathButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pathLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>291</width>
        <height>23</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
